homeassistant/components/samsungtv/config_flow.py

```diff
@@ -79,15 +78,69 @@ def _get_entry(self):
             data=data,
         )
 
+    async def _async_set_unique_id(self):
```

Comment:
Name this something more unique since its too easily confused with `async_set_unique_id`

Reply:
`_async_set_device_unique_id` makes it ?

Reply:
Updated